libs/gpu-dawn/sdk.zig

```diff
@@ -69,7 +69,7 @@ pub fn Sdk(comptime deps: anytype) type {
         }
 
         fn linkFromSource(b: *Builder, step: *std.build.LibExeObjStep, options: Options) !void {
-            // branch: generated-2022-08-06
+            // branch: origin/origin/generated-2023-01-12.1673546526
```

Comment:
```suggestion
            // branch: generated-2023-01-12.1673546526
```

@slimsag fix this in wrench (produced `origin/origin/generated-2023-01-12.1673546526` expected `generated-2023-01-12.1673546526`)